packages/flutter_tools/lib/src/ios/ios_workflow.dart

```diff
@@ -160,8 +165,19 @@ class IOSWorkflow extends DoctorValidator implements Workflow {
       ));
     }
 
+    if (hasCocoaPods) {
+      podStatus = ValidationType.installed;
+      messages.add(new ValidationMessage('CocoaPods version $cocoaPodsVersionText'));
+    } else {
+      podStatus = ValidationType.missing;
+      messages.add(new ValidationMessage.error(
+        'CocoaPods not installed; this is used for iOS development.\n'
+        'Install by running: \'sudo gem install cocoapods\''
```

Comment:
Given that we're using homebrew for other tooling, should we be recommending brew install `cocoapods` here?